mods/tuxemon/l18n/en_US/LC_MESSAGES/base.po

```diff
@@ -170,7 +170,9 @@ msgid "combat_state_lifeleech_get"
 msgstr "{user} is leeching health from {target}."
 
 msgid "combat_state_overfeed_get"
-msgstr "{user} Overfed {target}."
+
```

Comment:
just a comment, no need to change this, but please avoid changing indentation/white space in files unless you are correcting formatting.  in this case, an extra newline was added.